src/color-inline/README.md

```diff
@@ -42,14 +42,14 @@ Invalid color:
 
 | Name | Description |
 |------|-------------|
-| (default) | The element's main content—the color to be shown. Placed next to the color swatch. |
+| _(default)_ | The element's main content—the color to be shown. Placed next to the color swatch. |
 
 ### Attributes & Properties
 
 | Attribute | Property | Property type | Default value | Description |
 |-----------|----------|---------------|---------------|-------------|
 | `color` | `color` | `Color` &#124; `null` | - | The current color value. `null` for invalid colors. |
-| `value` | `value` | `string` &#124; `undefined` | - | The current value. `undefined` if the `value` attribute is not set. |
+| `value` | `value` | `string` &#124; `undefined` | - | The textual form of the color. Will have a value even if the color is invalid. `undefined` if the `value` attribute is not set. |
```

Comment:
That is incorrect, if the value attribute is not set you will still have a `value` property, with the current value.

Reply:
Oops. I experimented with the dev tools to check whether the `value` property was defined. I targeted the corresponding element and then tried `$0.value` in the console to see the result. Looks like it doesn't work this way, doesn't it?

Reply:
Oh `<color-inline>` doesn't use Props, so it doesn't follow the same standards. I'd consider it a bug.